drivers/hmis/app/models/hmis/project_group_criteria.rb

```diff
@@ -0,0 +1,169 @@
+###
+# Copyright 2016 - 2025 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+# frozen_string_literal: true
+
+module Hmis
+  class ProjectGroupCriteria < ::ModelForm
+    include ApplicationHelper
+    include ActionView::Helpers::TagHelper
+    include ActionView::Context
+    include ::Hmis::Concerns::HmisArelHelper
+
+    # Source data source for this project filter. Filter cannot be used across multiple data sources.
+    attribute :source_data_source_id, Integer, default: nil
+
+    # Attributes for filtering
+    attribute :project_ids, Array, default: []
+    attribute :organization_ids, Array, default: []
+    attribute :data_source_ids, Array, default: []
```

Comment:
Should this be:
```suggestion
    attribute :data_source_id, Integer, default: nil
```

The filter is tied to a single data source, so I'm not sure what it means to allow multiple here.  (Also the form appears to allow multiple selection of data sources.)